### PR TITLE
Working Gamma_Analysis

### PR DESCRIPTION
**Pull request by tybtab**
This is the working Gamma_Analysis that I received a couple weeks ago.

### CODE REVIEW

**Comment by bplimley** [comment]
No... if I merged this pull request we would lose all the improvements we have made in the last couple weeks. Our improvements are what I see in red that were removed in this commit. We need to figure out what Chris added to give proper output values, and just add/edit that specific code.

Here's how I would approach it. Chris' version is commit `833186a` on `tybtab:master`. Go through commits in the history of `bearing:master` (looking at merge commits on the black line in GitHub > Graphs > Network), doing a `diff` between `833186a` and a historical commit, until you find one with the minimal difference. For example the current `bearing:master` commit is `53b84ac`, so while on your `tybtab:master` branch you could do `git diff 53b84ac Gamma_Analysis.py` and you'll see all the differences in the file `Gamma_Analysis.py` (which in this case, are already shown in this pull request). But you want to go back a couple weeks in history to try to find where Chris' code diverged. Maybe `5219e55` and earlier.

When you find the least amount of difference between Chris' code and the historical commit, then that difference is what Chris changed. However, since we have restructured some things since then, you may have to take what he wrote and rewrite it slightly.

We can work on it Wednesday after the meeting if needed.

**Comment by bplimley** [comment]
There is also the possibility that Chris' code is just an old version of the code, and some of our changes have broken something. So we could approach it from a different angle of, what numbers and uncertainties are calculated in `peak_measurement`, and how those numbers and uncertainties get passed around, and why something gets set to 0.

**Comment by tybtab** [comment]
Agreed. Was only making this pull request to display the differences. I'll do what you suggested.

**Comment by bplimley** [comment]
Ah, okay, sorry. Thanks.

File: Gamma_Analysis.py [hunk 1]
```diff
@@ -60,20 +57,13 @@ def isotope_activity(isotope, emission_rates, emission_uncertainty):
     branching_ratio = isotope.list_sig_g_b_r
     activity = []
     uncertainty = []
-    weight = []
-    squares_total = []
     for i in range(len(branching_ratio)):
-        activity.append(emission_rates[i]/branching_ratio[i])
```
Comment (tybtab) [inline Gamma_Analysis.py:66]: This line is missing a factor of .01

File: Gamma_Analysis.py [hunk 2]
```diff
@@ -60,20 +57,13 @@ def isotope_activity(isotope, emission_rates, emission_uncertainty):
...
-        uncertainty.append(emission_uncertainty[i]/branching_ratio[i])
```
Comment (tybtab) [inline Gamma_Analysis.py:67]: Missing a factor of .01

**Comment by tybtab** [comment]
I fixed what was missing

**Comment by tybtab** [comment]
I'll do another pull request
closing this one

**Pull request closed without merging**